sentry_sdk/worker.py

```diff
@@ -186,3 +187,134 @@ def _target(self) -> None:
             finally:
                 self._queue.task_done()
             sleep(0)
+
+
+class AsyncWorker(Worker):
+    def __init__(self, queue_size: int = DEFAULT_QUEUE_SIZE) -> None:
+        self._queue: Optional[asyncio.Queue[Any]] = None
```

Comment:
is there a reason you set queue to None and instantiated later? I would slightly prefer having an empty queue already and getting rid of all the `queue is None` checks.

Reply:
hmm I see.. ok leave it for now then. I just prefer less branches in general.